peripheralmanager.go

```diff
@@ -108,6 +110,13 @@ func (pm PeripheralManager) StartAdvertising(ad AdvData) {
 	}
 
 	C.cb_pmgr_start_adv(pm.ptr, &cad)
+
+	BTPeripheralManagerDidStartAdvertising(pm.ptr, &err)
+	if err.code != 0 {
+		return errors.New("")
```

Comment:
Did you mean to populate this error with some message instead of just a empty string?

Reply:
right! will change that!

I'm actually not sure whether this is the better option - an exported isAdvertising might be better (as, this check might fail... at least init took almost 100ms! failed to advertise before that :) ). for now I'll push with an error, and this can be discussed further